models/collection.js

```diff
@@ -2,6 +2,9 @@
 
 const Joi = require('joi');
 const mutate = require('../lib/mutate');
+const PIPELINE_MODEL = require('./pipeline').get;
```

Comment:
non-blocking, but could you alphabetize the consts and not include extra newlines please :)